core/openapi.yaml

```diff
@@ -343,58 +326,58 @@ components:
           type: number
           format: double
           example: 0.25
-        timetable_gamma:
+        gamma:
           type: number
           format: double
           example: 0.5
-        tractive_effort_curve:
+        loading_gauge:
+          type: string
+          enum:
+            - G1
+            - G2
+            - GA
+            - GB
+            - GB1
+            - GC
+            - FR3_3
+            - FR3_3_GB_G2
+        features:
           type: array
           items:
-            type: object
-            properties:
-              speed:
-                type: number
-                format: double
-                minimum: 0
-              max_effort:
-                type: number
-                format: double
-                minimum: 0
-            example:
-              speed: 20
-              max_effort: 400000
-    RollingResistanceDavis:
+            type: string
+            example: "protecting systems"
+        power_class:
+          type: string
+          example: "powerClass"
```

Comment:
`example` should contain an example json value, not the description

Reply:
fixed